### ISSUE TITLE
When I send scopt an empty args list, it passes my immutable data structure through

### ISSUE THREAD

**Issue by rishter** [scopt/scopt#59]
This only happens when I have one or more cmd's defined, and neglects to fail or print usage

**Comment by dlipovetsky** [comment]
Agreed--I expected scopt to print usage if none of the one or more defined commands were specified as an argument.
